### PR TITLE
clickhous, chronusdb *: Add new features and rebuild the docs-structure

### PR DESCRIPTION
**Pull request by molliezhang**
### What type of PR is this?
 
/Add Features
/Optimize

### Which issue(s) this PR fixes?

NA

### What this PR does /  why we need it ?

Summary:

1. Add 1.0.7 version features in chronusdb
2. Add 1.1.5 and 1.1.6 version features in clickhouse
3. Rebuild the docs-structure
4. Optimize sentence in intro, manual, quickstart, index

### Special notes for your reviewer?
NA

/assign @pethu1110

### CODE REVIEW

File: content/database/chronusdb/manual/backup_restoration/enable_backup.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    <img src="../../../_images/backup_list.png" alt="备份列表" style="zoom:100%;" />
+
+备份创建成功后，可从备份节点创建新集群，恢复集群信息。
+
+![恢复集群](../../../_images/restore.png)
+
+> **注意**
+> 
+> 使用 Page cache 与最终一致模型，通常情况下不推荐使用备份功能。
```
Comment (pethu1110) [inline content/database/chronusdb/manual/backup_restoration/enable_backup.md:57]: 注意中的内容，表述不太清晰。

File: content/database/chronusdb/manual/config_para/check_para.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+draft: false
+---
+
+
+
+在 AppCenter 集群管理控制台，支持查看当前集群基本参数。
+
+> **说明**
+> 
+> 暂不支持在线修改 ChronusDB 集群的配置参数。
```
Comment (pethu1110) [inline content/database/chronusdb/manual/config_para/check_para.md:16]: 不要有“暂”这个词

File: content/dwh_bi/clickhouse/manual/backup_restoration/delete_backup.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+draft: false
+---
+
+
+
+在 AppCenter 集群管理控制台，ClickHouse 提供数据备份管理，可以通过**备份**服务来管理集群数据。
+
+若备份链接已过期或不再需要，可删除备份链或备份节点，节省资费。
+
+> **注意：**
```
Comment (pethu1110) [inline content/dwh_bi/clickhouse/manual/backup_restoration/delete_backup.md:16]: 注意后不加冒号

File: content/dwh_bi/clickhouse/manual/config_para/check_para.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+draft: false
+---
+
+
+
+在 AppCenter 集群管理控制台，支持查看当前集群基本参数。
+
+> **说明**
+> 
+> 暂不支持在线修改 ClickHouse 集群的配置参数。
```
Comment (pethu1110) [inline content/dwh_bi/clickhouse/manual/config_para/check_para.md:16]: 不要使用“暂”这个词

File: content/dwh_bi/clickhouse/manual/node_lifecycle/create_node.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+
+ClickHouse 集群节点，数据库性能越好，可根据需要增加集群节点。
+
+- 为了尽快的完成增加节点操作，让计算资源更快应用集群节点，增加节点过程不会迁移原有数据。
+
+- 若需对已有数据进行重新迁移，可以进入数据库执行 `SYSTEM RESHARDING` ，可对集群的所有节点进行数据重分布。
+
+   > **注解**：
```
Comment (pethu1110) [inline content/dwh_bi/clickhouse/manual/node_lifecycle/create_node.md:17]: 把“注解”替换为“说明”，后面不加冒号

Comment (molliezhang) [reply content/dwh_bi/clickhouse/manual/node_lifecycle/create_node.md:17]: done

File: content/dwh_bi/clickhouse/manual/config_para/check_para.md (hunk 4 above)
Comment (molliezhang) [reply content/dwh_bi/clickhouse/manual/config_para/check_para.md:16]: done

File: content/dwh_bi/clickhouse/manual/backup_restoration/delete_backup.md (hunk 3 above)
Comment (molliezhang) [reply content/dwh_bi/clickhouse/manual/backup_restoration/delete_backup.md:16]: done

File: content/database/chronusdb/manual/config_para/check_para.md (hunk 2 above)
Comment (molliezhang) [reply content/database/chronusdb/manual/config_para/check_para.md:16]: done

File: content/database/chronusdb/manual/backup_restoration/enable_backup.md (hunk 1 above)
Comment (molliezhang) [reply content/database/chronusdb/manual/backup_restoration/enable_backup.md:57]: done

**Review by pethu1110 — APPROVED** [review]

**Pull request merged**